gradle.properties

```diff
@@ -1,2 +1,2 @@
-springBootVersion=3.3.1
+springBootVersion=3.3.2
 springDataBomVersion=2024.0.1
```

Comment:
The version for 'spring-data-bom' should also be applied.

https://github.com/spring-projects/spring-data-bom/releases/tag/2024.0.2

Reply:
Thx